src/components/Articles/ResolutionDates.vue

```diff
@@ -127,6 +153,10 @@ export default class ResolutionDates extends Mixins(DateMixin, CommonMixin) {
     return (this.previousDates?.length > 0)
   }
 
+  get currentDateString (): string {
+    return this.dateToDateString(new Date())
```

Comment:
Are you sure the user's browser's date is set to Pacific timezone and is not correct (not incorrect or spoofed)?

Please use `getCurrentDate` instead. Please see how this is set to understand how to work around the possible date issues ^^^

Reply:
ah good point. Will do, thanks!

Reply:
Updated in next commit!